engine/src/main/java/org/terasology/engine/core/module/ModuleInstaller.java

```diff
@@ -38,7 +38,7 @@ public class ModuleInstaller implements Callable<List<Module>> {
     @Override
     public List<Module> call() throws Exception {
         Map<URI, Path> filesToDownload = getDownloadUrls(moduleList);
-        logger.info("Started downloading {} modules", filesToDownload.size());
+        logger.atInfo().addArgument(() -> filesToDownload.size()).log("Started downloading {} modules");
```

Comment:
Simple map size calculation once per module download.
```suggestion
        logger.info("Started downloading {} modules", filesToDownload.size()); //NOPMD
```